src/lucky_record/model.cr

```diff
@@ -86,4 +87,14 @@ class LuckyRecord::Model
     {% end %}
     {% FIELDS << {name: type_declaration.var, type: data_type, nilable: nilable.id, autogenerated: autogenerated} %}
   end
+
+  macro setup_fields_method
+    def self.fields
```

Comment:
Can you add a return type of `def self.fields : Array(Symbol)`? I know a lot of existing code doesn't do that, but I'd like to do it more so that the generated API docs are better

Reply:
Yepp, am new to crystal so have no real feeling for when to add types and when to infer them yet. ✅